enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -157,9 +206,9 @@ public static DbRepresentation createSomeData( GraphDatabaseService db )
     {
         try ( Transaction tx = db.beginTx() )
```

Comment:
Cluster has a coreTx(..) method which would be good to use here and in other places because it can cope with leader-switches. This is one of the main reasons for tests being flaky.

Reply:
Addressed